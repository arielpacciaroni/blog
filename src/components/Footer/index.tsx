export default function Footer() {
  return (
    <nav className="mt-auto py-16 bg-stone-800">
      <div className="section-wrapper">
        <div className="flex justify-between gap-4">
          <div>
            <h3 className="text-gray-400 select-none">Thanks!</h3>
            <p className="text-sm">
              with all ❤️ by{' '}
              <a
                href="https://www.linkedin.com/in/arielpacciaroni/"
                target="_blank"
                rel="noopener noreferrer"
                className="text-gray-100 hover:text-white"
              >
                Ariel
              </a>
            </p>
          </div>
          <div>
            <h3 className="text-gray-400 select-none">Follow me</h3>
            <ul>
              <li>
                <a
                  href="https://www.linkedin.com/in/arielpacciaroni/"
                  target="_blank"
                  rel="noopener noreferrer"
                  className="text-gray-100 hover:text-white text-sm"
                >
                  LinkedIn
                </a>
              </li>
              <li>
                <a
                  href="https://github.com/arielpacciaroni"
                  target="_blank"
                  rel="noopener noreferrer"
                  className="text-gray-100 hover:text-white text-sm"
                >
                  GitHub
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </nav>
  )
}
