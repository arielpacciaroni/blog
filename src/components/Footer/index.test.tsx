import { render } from '@testing-library/react'
import Footer from '.'
import '@testing-library/jest-dom'

describe('Footer', () => {
  it('renders footer unchanged', () => {
    const { container } = render(<Footer />)
    expect(container).toMatchSnapshot()
  })
})
