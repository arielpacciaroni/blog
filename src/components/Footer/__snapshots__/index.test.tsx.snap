// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders footer unchanged 1`] = `
<div>
  <nav
    class="mt-auto py-16 bg-stone-800"
  >
    <div
      class="section-wrapper"
    >
      <div
        class="flex justify-between gap-4"
      >
        <div>
          <h3
            class="text-gray-400 select-none"
          >
            Thanks!
          </h3>
          <p
            class="text-sm"
          >
            with all ❤️ by
             
            <a
              class="text-gray-100 hover:text-white"
              href="https://www.linkedin.com/in/arielpacciaroni/"
              rel="noopener noreferrer"
              target="_blank"
            >
              Ariel
            </a>
          </p>
        </div>
        <div>
          <h3
            class="text-gray-400 select-none"
          >
            Follow me
          </h3>
          <ul>
            <li>
              <a
                class="text-gray-100 hover:text-white text-sm"
                href="https://www.linkedin.com/in/arielpacciaroni/"
                rel="noopener noreferrer"
                target="_blank"
              >
                LinkedIn
              </a>
            </li>
            <li>
              <a
                class="text-gray-100 hover:text-white text-sm"
                href="https://github.com/arielpacciaroni"
                rel="noopener noreferrer"
                target="_blank"
              >
                GitHub
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </nav>
</div>
`;
